docs/examples/3dflex-custom-latent-trajectory.ipynb

```diff
@@ -247,42 +220,57 @@
xPxvbgEtm77H/zEHgGtAjxPGBKudbeJV4hrmGeOhG68Aw/kcraMqBBiDJs+JUdd9n/egbcmvfpUOu/XW9Adr1hRlgY/1b9h55/RzrNZ7751++5znpKPWrUs7PsMt7fUfLO8ff+97U19LujyMnjvuVRFEM14qYnQB1xW9zjqti7jWXMc5eqPo5YEihrpZU6yluGfH8BHqYE2xlzAf2NPMYM58x/rMXI97Ku27FpgPtKXVHyWTaz/umyqb3B+hmT2SdZ+HjJAYl7wIzE37mO81eveoqKAcdeExAq28b1x4SQx6HlrF+cH5w/d6+jBL6HsM+bKvrHUBuX11VjEm0KZCNcbc95n/ls1phYZxrnsdhYbF8M4A0McfpQEbjM/DadQwAPRpcHVG6xwWYfvAaN2NWdzR1Csolbihlw79JLTiTXW0ucIrYCGMGOdqtloSeWmlzPtBGYDTOeecU8VEj/KJXgoIRQgwzEkEPIQ/Aa1ACYHHeHBdcxWQKGtW65isywRHuktGi4fWFwGIcYUIm/QvKgEUoLVEIdgSLgB/vIbIrOp1QFkre7Seaw2CDmmO9CHcRQsOgi1CM7/pV8zirrdD3Tj0cTGmrO7U/K0gnPMt8jhvM3d3hx+CPXjLOAMASGKIUEt7CLHUybsAVcApbep+HQX2eLc5bTPuB919d1p10UXVVV/x88S++6aL3vWu9C9b7mRumqc5j3IrN/8jDOOxkSeJM9mdrsICdMc6eltQD/MYV2j6fOyxx1bjSdmbb765mv9YM7WCekVgTnccD+aq3g/wl7XD3kSdhmhAN3103pgwq23dMi6ME+Nn2Mko63wS77h+9txtt3Tw2rXpzNWrJ1FtYx3cxX7ZqlUjt6HXBxVEkKlCyb0mjmOb14K5PwRljCc3zZi0DFCyZs2aau8ksZzW+zj3oMV5Hq350XXd/Y11x9xhbTH2WsHxyDJ5oPuDHli0TVnDiJpc8qmb+Rk9UqATF3dp1xNHZZ0DoaLJvYHvqcucISoz+V6Fox4LhgWZxZ11YhiS+TbYUw3tiaFd9EXvIcYAHugVQFta4CO/HTOvOc2zx/eZXF2eckPSuH/j5gDQ+8ys+rIDNhifh9OoYQDo0+DqjNY5LMLugZnGPejdrU6+RN0BjzARXWp17wbsIbTpJlkXjzoJgB7j/Pmb5EFmjxcA5q6ZAlYEpGjRRQjjO6wyUdAFBOZxynBX93EVEwhzxvRSj5ZwgSGgSjdh3sdtkTZJAAe/eFf38iZBOweBxhTG7yPwRQGCkEg71A2gA6AbtkC7jB9gy4z7dW3rGt4VVywIii6axp1HYZv6SuJhc1qo3zlG1mq8AASx9NWr7Zh3Cr98HwEOQrLxtQrrR95xR3rLJZdsE6/88ItfnP7x3e9Ov91zzwrItCkq4rPcY0TLNfSRJPTUU0+ds/DdeOONFaBWgI9Z+k0M6HxT+UGfAOjc461yy/hXs3bDOxURjlvT+KnAsn4VT/DafAW6wDNfqbckXp5+m3kcxRjzfyE/9K+6537TpvSfLrgg7ZIpYyZJ24Zddkkf/uAHf3cF34gf1gljwNjyk7uFR4t2GzgXWAv48nkoeezxuKIzh5yTEdg6x1UOUJ8u8HE9Qwtj77WFlGMusyeybvMP3lbUo+WZvYo26tzxeZc9jXmFkuGkk06q2vEs4MYU1pChMNSlEllPEt5nHvC/7vL87V6ua3z0VPAZ3wFm6QsKDWg2nwr1euUjdMJn3fzhHXTwo2Wd71hLegsZYuK+bpvkraGuUUF0Sa6ZccD/iNN7Zl8bAPr4QzNgg/F5OI0aBoA+Da7OaJ3DIiwbGISdr371q9XhjGDNYQuYXGzZVHWFR5BDIDIxEN8D/uI1QAh68boZLYpwTJDc5eLe5RVQZ0Hn7nWt3rqP2zY0G5eOezCCDzQCNhCUBLDHH398JSB+5zvfqZLGmTRIyzb16FZufDt9RzDUYot1R0FWgBWz3QPQtZwi5Akc20BwHUAUDEdLuQI5Wdqhx6z5ADvoxIILWIIeBEra5H/GLFpcFfq1INXNdi3bWvagUeUH4wufqZM8BvEKJcrxjgqMJoCh5U0wwnvmMtDahfIDF1XHhLboD/97hVV0MWV+qMg4/pZb0ulXXbVN1+5fsiRdfO65afOWrNVmK8+9G6BHfjsGEfBCv/MYIRige/bZZ8+BfQA6Aj5zgHICCnmuOz71aD2Hr4B85qtzgv5cdtllVWgJdXlXPDw29rfOa0FFk1Y9+mLWdwAC+xT8Yq9ivTPHVSZFl1z5EMcReo2jFXx1AcmyHbWslOAxrinoYRxO/vSnqwzu0/x86IILqmvbSj51a5vv4Ds8g+9atV1zZh1nrncpz6CB8WBMnYdnnXVWtQ4i0M4zkauscZ8x6SD1CfjZU12f/K07vlcjQj9rlnkVQyfss9fIMXfphyFAKiZ4X4s47VKvCdyoD+DqWcBa8nYOFZ5modeSz/6EcgtlGTTRrgAdwM6+SX9QALo/6nnAM/Y0LOfQbagQHlHQSA4T2qdtk5LSdz7wjrGiHLyDDjPlqzSU3+6jKAKOOeaYytuKvcw5XTKnLNMVogZfxnWf70PPrJcdAPr4IzRgg/F5OI0aBoA+Da7OaJ3DIiwbGDd8hAiuz+LQVaAtq2E2SsUYNoQNrNXGsgL+EEgQXhA+SBIn6OB7kx/Zk7YkcX1i5aLwAV8RqBCCtExoFdQChbCmBUOFAgIf7/A+z3B5RHDCeu71RPRPgZG+69Zu8iYtJAJ2BDkVA4w79arQUAD1qrQIoOK8KAEzgkEtTr4DXQh2tItFuUkIi0oQkiwxpiafqnO1jzNRCx99BkDw0Xqm2zQ89eopytDXeFWWoLoNoKsQUUhXSKUN+ikI1wqtpYo6jUGHNgRn5yuw+uRrr02vu+mmbRbX+mXL0mUrV6bNO+9cPYtKgghyBbfRsgjPBCgCKd7RlRhw9IY3vKGK3WU+oBSBLq+pgl4tzc43kxXyzGu2vJse+mifNgEN7DXM
```

Comment:
Looks like there was some error when this notebook ran prevents the interactive widget from showing up in the [preview](https://cryosparc-tools-git-fork-plethorachutney-cu-42f17c-structurabio.vercel.app/examples/3dflex-custom-latent-trajectory.html):

```
Unhandled message type set_device_pixel_ratio. {'device_pixel_ratio': 1.2000000476837158, 'type': 'set_device_pixel_ratio'}
```

Not sure what to do about this. Are we ok with pushing this with an omitted trajectory plot?
![image](https://github.com/cryoem-uoft/cryosparc-tools/assets/1693461/635041e3-1afb-49d9-ad1e-b7686d5cf23e)